jgrapht-core/src/main/java/org/jgrapht/alg/KuhnMunkresMinimalWeightBipartitePerfectMatching.java

```diff
@@ -122,6 +122,34 @@ public double getMatchingWeight()
 
         return weight;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Matching<E> computeMatching()
+    {
+        Set<E> m = getMatching();
+        double w = 0d;
+        for (E e : m) {
+            w += graph.getEdgeWeight(e);
+        }
+        final double weight = w;
+        return new Matching<E>()
```

Comment:
This is the deprecated implementation, the new class uses directly the `DefaultMatching` class.